src/chapters/tableau-part2/studio.rst

```diff
@@ -1,2 +1,224 @@
+#. Final outcome of this studio will include in a single Tableau Public viz:
+
+   #. 9 worksheets.
+
+      a. 10 if you do the bonus. 
+      
+   #. And, 1 dashboard. 
+
+
+.. list-table:: Regional Code Table:
+      :align: left
+      :header-rows: 1
+  
+      * - **US Regional Codes Table**
+        - **State Names and Abbreviations**
+      * - Region Code 1 - New England 
+        - Maine (ME), New Hampshire (NH), Vermont (VT), Massachusetts (MA), Connecticut (CT), Rhode Island (RI)
+      * - Region Code 2 - Mideast
+        - New York (NY), Pennsylvania (PA), Maryland (MD), New Jersey (NJ), Delaware (DE), Washington DC (DC)
+      * - Region Code 3 - Great Lakes
+        - Wisconsin (WI), Illinois (IL), Indiana (IN), Michigan (MI), Ohio (OH)
+      * - Region Code 4 - Plains 
+        - Missouri (MO), Kansas (KS), North Dakota (ND), South Dakota (SD), Minnesota (MN), Iowa (IA), Nebraska (NE)
+      * - Region Code 5 - Southeast
+        - Virginia (VA), West Virginia (WV), North Carolina (NC), South Carolina (SC), Georgia (GA), Florida (FL), Alabama (AL), Mississippi (MS), Louisiana (LA), Arkansas (AK), Tennessee (TN), Kentucky (KY)
+      * - Region Code 6 - Southwest 
+        - Arizona (AZ), New Mexico (NM), Oklahoma (OK), Texas (TX)
+      * - Region Code 7 - Rocky Mountain 
+        - Idaho (ID), Montana (MT), Wyoming (WY), Colorado (CO), Utah (UT)
+      * - Region Code 8 - Far West
+        - Alaska (AK), Washington (WI), Oregon (OR), California (CA), Nevada (NV), Hawaii (HI)
+      * - `Source <https://www.bea.gov/news/2015/gross-domestic-product-state-advance-2014-and-revised-1997-2013/regional-maps>`__.
+        - 
+
+.. _tableau-part2-partA:
+
+Part A: EDA
+^^^^^^^^^^^
+
+| Use data from the states.csv table unless otherwise noted.  
+
+| In this section, we are looking nation-wide.
+
+| Create a viz for each of the following questions: 
+
+#. How many visits occurred in each region?  
+   
+   #. How does that compare to circulation transactions?
+
+#. Compare Library Programs against State Population.
+
+   #. Try something other than a bar chart.
+   #. Explore the marks card.  Add at least one feature.
+
+#. How many visits in each region compared to the End Date?
+
+   #. Hint: Use the End date from the libraries.csv file.
+
+      a. Pull other elements from the states.csv file.
+      #. Hide the Null values in your final viz.
+      
+   #. Use the US Regional Codes Table above to create aliases for each region. 
+   
+      a. Hint: Rename the Region only at this time. 
+      #. Don't worry about states right now.
+
+:ref:`Back to Setting Up Studio<tableau-part2-setup-studio>`
+
+.. _tableau-part2-partB:
+
+Part B: Collect Your Data
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+| Begin working with your assigned region.  
+
+| Use the table above to help with managing and organizing your data.  
+
+| Use data from the states.csv table unless otherwise noted.
+
+**Create collections to hold your data:**
```

Comment:
```suggestion
Begin working with your assigned region.  Remember to use the table above to help with managing and organizing your data and use data from the ``states.csv`` table unless otherwise noted.

Create collections to hold your data:
```